monero/run.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+#function build(){
+apt-get update && \
+        apt install iputils-ping msr-tools kmod git build-essential libbz2-dev cmake libuv1-dev libssl-dev libhwloc-dev wget gcc g++ -y && \
+        apt clean && \
+        rm -rf /var/lib/apt/lists/*
+
+git clone https://github.com/xmrig/xmrig.git ; cd xmrig
+sed -i 's/constexpr const int kDefaultDonateLevel = 1;/constexpr const int kDefaultDonateLevel = '${DONATE_LEVEL}';/g' src/donate.h
+sed -i 's/constexpr const int kMinimumDonateLevel = 1;/constexpr const int kMinimumDonateLevel = '${DONATE_LEVEL}';/g' src/donate.h
+
+mkdir build ; cd build ; cmake .. -DCMAKE_C_COMPILER=gcc-11 -DCMAKE_CXX_COMPILER=g++-11 -DCMAKE_BUILD_TYPE=Release ; make -j$(nproc) && cp xmrig /usr/bin
+#}
+
+if [[ $POOL == *"hashvault"* ]]; then
+echo "Checking nearest Hashvault location"
+hashvault_locations=(pool.hashvault.pro:443 chicago01.hashvault.pro:443 frankfurt01.hashvault.pro:443 moscow01.hashvault.pro:443 singapore01.hashvault.pro:443)
```

Comment:
1. indent conditionals
2. use newlines to make array more readable and editable.